<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>363</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="title">
     <string>日期时间(QTime QDate QDateTime)</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>391</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout" rowstretch="0,0,0,0" columnstretch="0,0,0">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="spacing">
       <number>10</number>
      </property>
      <item row="1" column="2">
       <widget class="QLineEdit" name="lineEdit"/>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit">
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QLabel" name="label_showDateTime">
        <property name="text">
         <string>时间：</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QLineEdit" name="lineEdit_3"/>
      </item>
      <item row="3" column="1">
       <widget class="QDateTimeEdit" name="dateTimeEdit"/>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_4">
        <property name="text">
         <string>日期时间</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="2">
       <widget class="QPushButton" name="pushButton_getTime">
        <property name="text">
         <string>获取当前日期时间</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>日期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="lineEdit_2"/>
      </item>
      <item row="2" column="1">
       <widget class="QDateEdit" name="dateEdit">
        <property name="displayFormat">
         <string>yyyy/MM/dd</string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>10</y>
      <width>341</width>
      <height>311</height>
     </rect>
    </property>
    <property name="title">
     <string>日历组件(Calendar Widget)</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>341</width>
       <height>291</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1,100" columnstretch="2,12">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>5</number>
      </property>
      <item row="2" column="0">
       <widget class="QLabel" name="label_7">
        <property name="text">
         <string>选择日期</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QCalendarWidget" name="calendarWidget"/>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="lineEdit_showDate"/>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>391</width>
      <height>151</height>
     </rect>
    </property>
    <property name="title">
     <string>定时器(Timer)</string>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>391</width>
       <height>131</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="2">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_6">
        <property name="text">
         <string>定时周期</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLCDNumber" name="lcdNumber_h"/>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="pushButton_begin">
        <property name="text">
         <string>开始</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="3">
       <widget class="QProgressBar" name="progressBar_timer">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLCDNumber" name="lcdNumber_s"/>
      </item>
      <item row="2" column="1">
       <widget class="QLCDNumber" name="lcdNumber_m"/>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="pushButton_stop">
        <property name="text">
         <string>停止</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QPushButton" name="pushButton_period">
        <property name="text">
         <string>设置周期</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QTimeEdit" name="timeEdit_inputTimer">
        <property name="displayFormat">
         <string>HH:mm:ss</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>761</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
